File: app/src/main/java/com/adenilson/testeandroid/networking/webservices/investiment/MoreInfoResponse.java
package com.adenilson.testeandroid.networking.webservices.investiment;

import com.google.gson.annotations.SerializedName;

/**
 * @author: Adenilson N. da Silva Junior
 * On date: 09/06/2018
 */

public class MoreInfoResponse {

    private MoreInfoDetailResponse month;
    private MoreInfoDetailResponse year;
    @SerializedName("12months")
    private MoreInfoDetailResponse twelveMonths;

    public MoreInfoDetailResponse getMonth() {
        return month;
    }

    public MoreInfoDetailResponse getYear() {
        return year;
    }

    public MoreInfoDetailResponse getTwelveMonths() {
        return twelveMonths;
    }

}

File: app/src/main/java/com/vctapps/santanderchallenge/asset/data/entity/MoreInfoResponse.kt
package com.vctapps.santanderchallenge.asset.data.entity

import com.google.gson.annotations.SerializedName

data class MoreInfoResponse(@SerializedName("month")    val month: MoreInfoDetailResponse?,
                            @SerializedName("year")     val year: MoreInfoDetailResponse?,
                            @SerializedName("12months") val twelveMonths: MoreInfoDetailResponse?)